.github/workflows/ghostv5.yml

```
@@ -182,6 +182,37 @@
           username: ${{ needs.myvars.outputs.GITHUB_ORG }}
           password: ${{ github.actor == 'dependabot[bot]' && secrets.TOKEN_GPR || secrets.TOKEN_GPR }}
 
+      - name: Generate cache keys
+        id: cache-key
+        run: |
+          # Extract versions from Dockerfile
+          GHOST_VERSION=$(grep 'ARG VERSION=' ${{ env.SUB_DIR }}/Dockerfile | head -n 1 | cut -d'"' -f2)
+          GHOST_CLI_VERSION=$(grep 'ARG GHOST_CLI_VERSION=' ${{ env.SUB_DIR }}/Dockerfile | head -n 1 | cut -d'"' -f2)
+          NODE_VERSION=$(grep 'ARG NODE_VERSION=' ${{ env.SUB_DIR }}/Dockerfile | head -n 1 | cut -d'"' -f2)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)